app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Vaccination required to enter the cafe")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("Vaccine is expired. Need to renew it.")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("A mask is required to enter the cafe")
+        else:
```

Comment:
You have an unnecessary 'else' statement here. After your conditions in lines 14, 16, and 18, you are raising an exception, which means if any of those conditions is true, the function execution will stop there. If none of those conditions is true, the function execution will continue to line 21 anyway, so there's no need for the 'else' statement. It's a good coding practice to avoid unnecessary 'else' statements.